pyOCD/gdbserver/gdbserver.py

```diff
@@ -489,7 +524,29 @@ def breakpoint(self, data):
             self.target.removeWatchpoint(addr, size, watchpoint_type)
         return self.createRSPPacket("OK")
 
-    def resume(self):
+    def stopReasonQuery(self):
+        # In non-stop mode, if no threads are stopped we need to reply with OK.
+        if self.non_stop and self.is_target_running:
+            return self.createRSPPacket("OK")
+
+        return self.createRSPPacket(self.target.getTResponse())
+
+    def _get_resume_step_addr(self, data):
+        if data is None:
+            return None
+        data = data.split('#')[0]
+        if ';' not in data:
+            return None
+        # c[;addr]
+        if data[0] in ('c', 's'):
+            addr = int(data[2:], base=16)
+        # Csig[;addr]
+        elif data[0] in ('C', 'S'):
+            addr = int(data[1:].split(';')[1], base=16)
+        return addr
+
+    def resume(self, data):
+        addr = self._get_resume_step_addr(data)
```

Comment:
It doesn't look like anything is being done with addr here and in step.  Do you have future plans for this?